audio-analysis/lib/etl/dfHelper.py

```diff
@@ -24,3 +24,9 @@ def customLeftJoin(dfLeft, dfRight, joinCondition):
             joinedDf.set_value(index, 'amplitude_peak', True)
 
     return joinedDf
+
+def checkJoin(dfLeft, dfRight, joinDescription):
```

Comment:
Maybe we could make this return true or false so we can stop if the join didn't happen correctly?

Reply:
agreed.